allink_core/core_apps/allink_teaser/templates/allink_teaser/includes/_link.html

```diff
@@ -4,6 +4,8 @@
     <div class="teaser__link">
         {% if instance.softpage_enabled %}
```

Comment:
this if/ else statement is probably obsolete, since you added the properties "link_icon" and "link_attributes" on the model, right?